recipes/ruby/all/conanfile.py

```diff
@@ -1,22 +1,17 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.apple.apple import is_apple_os, to_apple_arch # FIXME: private conan API?
```

Comment:
```suggestion
# TODO: Update to conan.tools.apple after Conan 1.51.3
from conans.tools import is_apple_os, to_apple_arch # FIXME: private conan API?
```
There is a hotfix running now: #12341

Reply:
I guess #12341 will be merged first, I'll rebase after that.